pro_tes/middleware/middleware_pipeline.py

```diff
@@ -0,0 +1,128 @@
+"""Set up the middleware pipeline."""
+import importlib
+import logging
+
+from flask import current_app
+
+logger = logging.getLogger(__name__)
+
+
+class MiddlewarePipeline:
+    """Middleware pipeline."""
```

Comment:
Can you please document args, attributes, return values and raised errors to the class, its methods and any static functions in the module?

Also, extend descriptions where necessary to facilitate understanding of the code, e.g., what is the class good for.

Reply:
yeah sure, will fix in the next commit